.idea/codeStyles/Project.xml

```diff
@@ -67,6 +67,8 @@
       </indentOptions>
     </codeStyleSettings>
     <codeStyleSettings language="kotlin">
+      <option name="CALL_PARAMETERS_WRAP" value="0" />
+      <option name="CALL_PARAMETERS_LPAREN_ON_NEXT_LINE" value="false" />
```

Comment:
Not sure why we need this now. I'm attributing this to latest EAP but something to keep an eye on